standalone/inc/pup/PUPTrigger.h

```diff
@@ -19,17 +19,39 @@ typedef enum
    PUP_TRIGGER_PLAY_ACTION_CUSTOM_FUNC     // Call a custom function
 } PUP_TRIGGER_PLAY_ACTION;
 
+struct StateTrigger{
+   string m_sName;
+   int value;
+};
+
+const string NO_TRIGGER = "";
+
 const char* PUP_TRIGGER_PLAY_ACTION_TO_STRING(PUP_TRIGGER_PLAY_ACTION value);
 
 class PUPTrigger
 {
 public:
-   ~PUPTrigger() {}
-
+   ~PUPTrigger() { }
    static PUPTrigger* CreateFromCSV(PUPScreen* pScreen, string line);
    bool IsActive() const { return m_active; }
    const string& GetDescription() const { return m_szDescript; }
-   const string& GetTrigger() const { return m_szTrigger; }
+   /**
```

Comment:
Any class methods that have lots of code, I'd rather have in the cpp instead of the h file. Especially since you have lots of comments about the purpose of the function.

if you wanted to stay in, you could do something like:

```
const string& GetMainTriggerName() const { return !m_triggers.empty() ? m_triggers.front().m_sName : NO_TRIGGER; }
```